backend/tests/test_main.py

```
@@ -0,0 +1,91 @@
+import sys
+import os
+sys.path.append(os.path.dirname(os.path.dirname(os.path.abspath(__file__))))
+
+import pytest
+from fastapi.testclient import TestClient
+from main import app
+from database.session import engine, Base
+
+@pytest.fixture
+def client():
```

Ruff (0.8.2)

Missing return type annotation for public function `client`

(ANN201)